docs/rdbms/naming-convention.md

```diff
@@ -0,0 +1,134 @@
+---
+id: rdbms-naming-convention
+title: RDBMS Naming Convention
+sidebar_label: Naming Convention
+---
+
+#### The following convention should be followed for naming RDBMS:
+
+* Use `snake_case` with lowercase while naming.
```

Comment:
* If the name of the table or column must consist of more than one word, use an underscore to connect them, for example `snake_case`.

Reply:
Daii I guess this is implied. :smile:

Reply:
Let's explicitly define it as well.